packages/types/package.json

```diff
@@ -12,6 +12,7 @@
     "!*.map"
   ],
   "scripts": {
+    "watch": "tsc --watch",
```

Comment:
this change is unrelated, revert it.

Reply:
I will revert this before the pull request pass all the CI pipeline, now i need this to debug at my local.